agent-lib/src/main/java/org/terracotta/angela/agent/AgentController.java

```diff
@@ -248,7 +248,7 @@ public boolean installVoter(InstanceId instanceId, TerracottaVoter terracottaVot
   }
 
   public boolean installClusterTool(InstanceId instanceId, String hostName, Distribution distribution,
-                                    License license, String kitInstallationName, SecurityRootDirectory securityRootDirectory,
+                                    String kitInstallationName, License license, SecurityRootDirectory securityRootDirectory,
```

Comment:
do we really need to change the parameter ordering ?

Reply:
No I will fix it